src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/FunctionId.cs

```diff
@@ -647,6 +647,8 @@ internal enum FunctionId
 
     Copilot_Rename = 851,
 
+    Copilot_AnalyzeChange = 860,
```

Comment:
`VSCode_LanguageServer_Started` is also 860

Reply:
yup!